src/dvc_data/hashfile/meta.py

```diff
@@ -17,6 +17,7 @@ class Meta:
     PARAM_VERSION_ID: ClassVar[str] = "version_id"
     PARAM_ETAG: ClassVar[str] = "etag"
     PARAM_CHECKSUM: ClassVar[str] = "checksum"
+    PARAM_MD5: ClassVar[str] = "md5"
 
     isdir: bool = False
```

Comment:
This may end up showing on `dvcfiles` as we just dump directly. Do we set it anywhere?

Reply:
We only use it in `dvc import/get` and don't dump it directly yet. Definitely keeping an eye out for this.